test/test_function_schema.py

```diff
@@ -112,6 +112,53 @@ def test_string_optional_parameter_default_value(self):
         schema_b = parse_schema(str(schema_a))
         self.assertEquals(schema_a, schema_b)
 
+    def test_forward_compatible_arguments_without_out(self):
+        old_schema = parse_schema('any(Tensor self, int a, int b=1) -> Tensor')
```

Comment:
Add comments for each case being tested before it's tested

Reply:
done